pallets/sum-storage/Cargo.toml

```diff
@@ -3,6 +3,22 @@ name = "sum-storage"
 version = "2.0.0-alpha.6"
 authors = ["Joshy Orndorff"]
 edition = "2018"
+description = "A pallet with two storage items whose sum is exposed via a custom runtime API"
+license = "GPL-3.0-or-later"
+homepage = "https://github.com/substrate-developer-hub/recipes"
```

Comment:
I think it's fine to omit this when there is no actual homepage.

Reply:
I was also wondering about that. @stiiifff recommended adding it before publishing. What do you think @stiiifff?

Reply:
Decided to omit it.